modules/ding_content_rss/ding_content_rss.install

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @file
+ * Handle module updates.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_content_rss_install() {
+  db_update('system')
+    ->fields(array('weight' => 10))
+    ->condition('name', 'ding_content_rss', '=')
+    ->execute();
+}
+
+/**
```

Comment:
Normally new modules to the DDB CMS don't have update hooks. But is this module already used in production on some site? If so I think we need to leave them here.

Reply:
Yes, they are already in use on some sites.

Reply:
👍